src/components/atoms/Icon/index.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable prettier/prettier */
+import cx from 'classnames';
+import React from 'react';
+import type { IconBaseProps } from 'react-icons/lib';
+
+export type IconProps = {
+  size?: number;
+  href?: string;
+  onClick?: () => void;
+  icon?: IconBaseProps ;
+  iconType?: 'primary' | 'pink' | 'success' | 'warning' | 'danger';
+};
+
+const Icon: React.FC<IconProps> = ({
+  href,
+  onClick,
+  icon,
+  iconType,
+  size,
```

Comment:
You can add default values to the props:

`size = 50,`

If it's null, it will use 50

Reply:
I've added the size = 50 on the props. Thank you.